masl.src.html

```diff
@@ -34,37 +33,38 @@ <h2>Introduction</h2>
       <h2>How to use a MASL document</h2>
       <p>
         The recommended structure for DASL metadata is to insert a dCBOR42 document "between" each
-        CID and its resource(s), essentially using dCBOR42 as headers wrapping a CID-addressed resource.
+        CID and its resource(s), essentially using dCBOR42 as headers wrapping one or more CID-addressed resource(s).
         To do this, simply replace the CID of a resource with the CID of a dCBOR42 document that contains either a
-        top-level (tag 42) property <code>src</code> (for a single resource), containing the
-        CID of the resource it describes, or a top-level mapping <code>resources</code> (for a collection
-        of resources), mapping paths to resource CIDs, each with optional dCBOR42 metadata ([[cid]], [[dcbor42]]).
+        top-level (tag 42) property named <code>src</code>.
+        To bundle a set of CIDs in a logical unit, drop the top-level <code>src</code> and replace it with a mapping of relative paths to CIDs (if free of metadata) or to dCBOR42 metadata objects including <code>src</code>.
       </p>
       <p>
         It is preferable to nest any metadata in a top-level object to namespace your own metadata
-        standard (inside an object named, for example, `my-cool-project-v1`) rather than opaque version-bits or magic numbers.
-        There are a few reserved words at the top-level, but these can be avoided by nesting any conflicts.
+        template (inside an object named, for example, `my-cool-project-v1`) rather than using opaque UUIDs, version-bits inserted into values, or magic numbers.
```

Comment:
```suggestion
        vocabularies (inside an object named, for example, `my-cool-project-v1`) rather than using opaque UUIDs, version-bits inserted into values, or magic numbers.
```